DataConnectors/AzureFunctionsManualDeployment.md

```diff
@@ -0,0 +1,35 @@
+# Function app manual deployment instructions 
+
+1. Deploy a Function App
+
+	NOTE: You will need to [prepare VS code](https://docs.microsoft.com/azure/azure-functions/functions-create-first-function-python#prerequisites) for Azure function development.
+
+	Download the [Azure Function App](https://aka.ms/sentinel-SlackAuditAPI-functionapp) file. Extract archive to your local development computer.
```

Comment:
This should be some generic placeholder, dont hard-code SlackAudit file

Reply:
Removed hard coded value